bundles/org.openhab.binding.roku/src/main/java/org/openhab/binding/roku/internal/handler/RokuHandler.java

```diff
@@ -320,6 +350,41 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR);
                 }
             }
+        } else if (channelUID.getId().equals(POWER)) {
```

Comment:
```suggestion
        } else if (POWER.equals(channelUID.getId())) {
```

Reply:
This is probably something that needs to be changed across all of the channels under handleCommand.  I used the original to keep it in line with the rest of them.

Reply:
I agree it's out of scope to fix existing and you don't have to do that, but there is no reason to "keep it in line with potential `NullPointerException`" for new code.